src/components/AdminProfile/SendLinkModal/SendLinkModal.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ChakraProvider,
+  Text,
+  Textarea,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalBody,
+  ModalCloseButton,
+  Button,
+} from '@chakra-ui/react';
+import Checkmark from '../../../assets/Check.png';
+import styles from './SendLinkModal.module.css';
+
+const ModalOne = ({ count, setCount }) => {
+  return (
+    <div className={styles['modal-content']}>
+      <Text fontSize="2xl" fontWeight="bold" textAlign="left">
+        Send registration link via email:
+      </Text>
+      <Textarea placeholder="ex: jdoeFYA@gmail.com" />
+      <Button onClick={() => setCount(count + 1)} color="white" bg="#345E80" iconSpacing="120px">
+        Send Link
+      </Button>
```

Comment:
Currently the button is centered, is it possible to mirror the Figma designs for the modal or to add some spacing between the text area and the button?